nms/packages/magmalte/Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM node:12.18-alpine as builder
+FROM node:14.19-alpine as builder
```

Comment:
Note that version 10 is still used below. See also https://github.com/magma/magma/pull/12206/commits/214925cbdc3ae74a4485bf72c21216b0fc7572c7